interpreters/src/create.rs

```diff
@@ -136,3 +139,44 @@ impl<C: Manager> Interpreter for CreateInterpreter<C> {
         self.execute_create().await.context(Create)
     }
 }
+
+pub fn get_catalog<C: Manager>(
```

Comment:
get_default_catalog

Reply:
It seems the two methods `get_catalog` and `get_schema` should be placed in `show.rs`.